pages/index.js

```diff
@@ -79,16 +71,16 @@ export default function HomePage() {
       {possibleFilters.map((filter) => {
         return (
           <button
-            key={filter.filterName}
+            key={filter}
             onClick={() => {
               handleFilterClick(filter);
             }}
             style={{
-              backgroundColor: filter.isClicked ? "black" : "",
-              color: filter.isClicked ? "white" : "black",
+              backgroundColor: activeFilters.includes(filter) ? "black" : "",
+              color: activeFilters.includes(filter) ? "white" : "black",
```

Comment:
```suggestion
              backgroundColor: activeFilters.includes(filter) && "black",
              color: activeFilters.includes(filter) ? "white" : "black",
```

also you have duplicate logic, try storing it in a separate variable.

```javascript
const filtered = activeFilters.includes(filter);

```

and ask for it.